src/accessory/SwitchAccessory.ts

```diff
@@ -44,6 +48,14 @@ export default class SwitchAccessory extends BaseAccessory {
     }
 
     configureOn(this, service, schema);
+    configureEnergyUsage(
```

Comment:
For the multiple switch, should we only add energy usage into the main switch?

Reply:
Not sure, I don’t have a multi switch available to play around with. Pretty sure it wouldn’t hurt to add it to both switches but you certainly know more how a multi switch is presented, so your call

Reply:
OK, I think I will only add to main switch to skip schema like `switch_1`, `switch_usb1`, `switch_backlight`.

Reply:
I've pushed a commit (5a510c5) that only includes the energy stats in the first switch. You can either pick that version or the revision before